engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-errata-modal.html

```diff
@@ -27,7 +27,14 @@ <h4 data-block="modal-header">Content Host Errata Management</h4>
       </div>
     </div>
 
-    <div data-extend-template="common/views/katello-agent-notice.html"></div>
+    <div ng-show="katelloAgentPresent" data-extend-template="common/views/katello-agent-notice.html"></div>
+    <div bst-feature-flag="remote_actions">
+      <p bst-alert="warning" ng-hide="hostToolingEnabled">
+        <span translate>
+          Remote execution is not present and katello-agent is not enabled. Performing host actions is disabled.
```

Comment:
This is tricky to word properly because we need to be clear that it's the server that's configured this way, not the client. Something like:

```
Performing host package actions is disabled because Katello is not configured for Remote Execution or Katello Agent
```

Reply:
@jturel addressed all feedback and put in a second commit for you :)